cfpb/consumerfinance.gov
# Form explainers: add breadcrumbs to template

Adds code to output breadcrumbs in form explainer template


## Additions

- Code to get breadcrumbs and output them in form explainer template




## How to test this PR

1. Check out branch
2. Load recent db dump
3. Navigate to [prepaid explainer page](http://localhost:8000/consumer-tools/prepaid-cards/understand-fees/) and verify that breadcrumbs are displayed


## Screenshots

| <img width="1214" alt="Screen Shot 2022-09-27 at 9 06 40 AM" src="https://user-images.githubusercontent.com/778171/192581792-a8eb45a4-6777-441b-a6db-1143095c99e1.png"> |
| -- |


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
